packages/pass-style/src/passStyleOf.js

```diff
@@ -33,7 +34,11 @@ import { assertPassableString } from './string.js';
 /** @import {PassStyleOf} from './types.js' */
 /** @import {PrimitiveStyle} from './types.js' */
 
-/** @typedef {Exclude<PassStyle, PrimitiveStyle | "promise">} HelperPassStyle */
+/**
+ * TODO will need a symbol helper too.
+ * @typedef { "byteArray" | Exclude<PassStyle, PrimitiveStyle | "promise">
+ * } HelperPassStyle
+ */
```

Comment:
```suggestion
/**
 * Non-primitive pass styles (except "promise") and special primitive pass
 * styles ("byteArray", and in the future also "symbol") are supported by
 * helpers.
 * @typedef {Exclude<PassStyle, PrimitiveStyle | "promise"> | "byteArray"} HelperPassStyle
 */
```
but for that matter, I don't think this HelperPassStyle type is carrying its own weight, and would prefer dropping it:
```js
/**
 * @template {Record<PassStyle, PassStyleHelper> & {remotable: PassStyleHelper}} HelpersRecord
 * @param {Array<PassStyleHelper & {styleName: keyof HelpersRecord}>} passStyleHelpers
 * @returns {HelpersRecord}
 */
const makeHelperTable = passStyleHelpers => {
```